docs/contribute/getting-started.md

```diff
@@ -30,301 +30,150 @@ page_nav:
-<td style="text-align: left"><code class="highlighter-rouge">chevron-up</code></td>
-<td style="text-align: left"><i class="icon icon--chevron-up" style="font-size: 25px;"></i></td>
-</tr>
-<tr>
-<td style="text-align: left">Home</td>
-<td style="text-align: left"><code class="highlighter-rouge">home</code></td>
-<td style="text-align: left"><i class="icon icon--home" style="font-size: 25px;"></i></td>
-</tr>
-<tr>
-<td style="text-align: left">Maximize</td>
-<td style="text-align: left"><code class="highlighter-rouge">maximize</code></td>
-<td style="text-align: left"><i class="icon icon--maximize" style="font-size: 25px;"></i></td>
-</tr>
-<tr>
-<td style="text-align: left">X (Close)</td>
-<td style="text-align: left"><code class="highlighter-rouge">x</code></td>
-<td style="text-align: left"><i class="icon icon--x" style="font-size: 25px;"></i></td>
-</tr>
-</tbody>
-</table>
-
-<h2 id="change-log">Change Log</h2>
-<p>All notable changes to this project will be documented here.
-This project adheres to <a href="http://semver.org/">Semantic Versioning</a>.</p>
-
-<h3 id="v100---2017-12-08">[v1.0.0] - 2017-12-08</h3>
-<p>Initial release.</p>
-
-<h2 id="credits">Credits</h2>
-<p>List of vendor assets we used to create this theme:</p>
-<ul>
-<li><a href="https://github.com/twbs/bootstrap">Bootstrap</a> (<a href="https://github.com/twbs/bootstrap/blob/master/LICENSE">MIT License</a>)</li>
-<li><a href="https://github.com/necolas/normalize.css">normalize.css</a> (<a href="https://github.com/necolas/normalize.css/blob/master/LICENSE.md">MIT License</a>)</li>
-<li><a href="http://jquery.com">jQuery</a> (<a href="https://tldrlegal.com/license/mit-license">MIT License</a>)</li>
-<li><a href="https://github.com/liabru/jquery-match-height">jquery-match-height</a> (<a href="https://github.com/liabru/jquery-match-height/blob/master/LICENSE">MIT License</a>)</li>
-<li><a href="https://github.com/simple-icons/simple-icons">simple-icons</a> (<a href="https://github.com/simple-icons/simple-icons/blob/develop/LICENSE.md">CC0-1.0 License</a>)</li>
-<li><a href="https://github.com/colebemis/feather">feather</a> (<a href="https://github.com/colebemis/feather/blob/master/LICENSE">MIT License</a>)</li>
-<li><a href="https://fonts.google.com/specimen/Montserrat">Montserrat</a> (<a href="http://scripts.sil.org/cms/scripts/page.php?site_id=nrsi&amp;id=OFL">SIL Open Font License, 1.1</a>)</li>
-<li><a href="https://fonts.google.com/specimen/Noto+Sans">Noto Sans</a> (<a href="http://www.apache.org/licenses/LICENSE-2.0.html">Apache License, version 2.0</a>)</li>
-<li><a href="https://fonts.google.com/specimen/Source+Code+Pro">Source Code Pro</a> (<a href="http://scripts.sil.org/cms/scripts/page.php?site_id=nrsi&amp;id=OFL">SIL Open Font License, 1.1</a>)</li>
-</ul>
-
-                </div><!-- /.content -->
-            </div><!-- /.col -->
-        </div><!-- /.row -->
-</div><!-- /.container -->
+## What is the SingularityNET Developer Portal Theme?
```

Comment:
I think discussing the theme is unnecessary. We should just say the site is built with Jekyll and it is compatible with being hosted on github or gitlab.

Reply:
Left just a small explanation and a note with the link to the Jekyll documentation.